### ISSUE TITLE
Doesn't launch in Steam Linux

### ISSUE THREAD

**Issue by kayahr** [friendlyfiregame/friendlyfiregame#354]
**Describe the bug**
Game doesn't start when launching from within Steam on Linux or Steam Deck.

**To Reproduce**
1. Install Friendly Fire in Steam on Linux or Steam Deck
2. Launch the game by clicking the Play button. The Play button switches to Stop but no game starts. Sometimes (rarely) the game manages to start after a long wait time. But most of the time it timeouts and the Stop button changes back into the Play button.

The game starts without problems when running the executable manually outside of Steam. It only has trouble when launching from within Steam. The same happens when adding Friendly Fire as a Non-Steam game and selecting the friendlyfiregame binary.

Also tried compiling latest version from source and replacing the game files in the Steam folder. Same result: Starts without problems manually but not from within Steam. Updating all dependencies also doesn't help.

**Execution environment (please complete the following information):**
 - Operating system: Debian 12 and SteamOS on Steam Deck OLED (Arch Linux)
 - Distribution variant: Steam

**Comment by headcr4sh** [comment]
I see...

The game seems to be working *sometimes* and *under certain circumstances* which is really a bad experience for Linux users. :-(

This issue might be related to:
* https://github.com/ValveSoftware/steam-runtime/issues/579
* https://github.com/ValveSoftware/Proton/issues/2818#issuecomment-1192329628
* https://github.com/ValveSoftware/steam-for-linux/issues/8031
* ...

Right now, I don't feel like rebuilding Electron myself.
Maybe a well-maintained, pre-packaged Steam-compatible Electron package already exists... haven't found one yet, though. :-(

**Comment by kayahr** [comment]
I wonder if this has something to do with the steamoverlay library which is preloaded before starting the game. When I do this manually before starting the game outside of Steam then I get the same behavior, Electron just freezes most of the time. But it doesn't help disabling the steam overlay in Steam. I guess the library is preloaded anyway, it is just not active then. And on Steam Deck the steam overlay can't be disabled at all.

**Comment by kayahr** [comment]
Found the problem by checking the command line arguments of other Electron games on Steam. They all use the `--no-sandbox` Electron parameter. When adding this parameter manually in Steam then the game launches correctly. Also works on Steam Deck. 

I added the parameter in steamworks for Linux. Works now on Linux and Steam Deck. 

I didn't add the parameter to the other operating systems because I can't test that and I don't know if the parameter is needed there at all.
